packages/financial-templates-lib/test/price-feed/EthVixPriceFeed.js

```diff
@@ -0,0 +1,174 @@
+const { ETHVIXPriceFeed } = require("../../src/price-feed/EthVixPriceFeed");
+const { NetworkerMock } = require("../../src/price-feed/NetworkerMock");
+const { parseFixed } = require("@uma/common");
+const moment = require("moment");
+const winston = require("winston");
+const { assert } = require("hardhat");
+
+describe("EthVixPriceFeed.js", () => {
+  let networker;
+  let priceFeed;
+  let inversePriceFeed;
+  let mockTime = 1616598000000;
+
+  const getTime = () => mockTime;
+  const minTimeBetweenUpdates = 60;
+
+  const historicalResponse = [
+    {
+      currency: "ETH",
+      timestamp: "2021-03-24T14:30:00.000Z",
+      spotPrice: "1737.04",
+      vix: "88.47",
+      iVix: "113.04"
+    },
+    {
+      currency: "ETH",
+      timestamp: "2021-03-24T14:45:00.000Z",
+      spotPrice: "1738.36",
+      vix: "88.78",
+      iVix: "112.62"
+    },
+    {
+      currency: "ETH",
+      timestamp: "2021-03-24T15:00:00.000Z",
+      spotPrice: "1740.26",
+      vix: "70.2",
+      iVix: "142.44"
+    }
+  ];
+
+  beforeEach(() => {
+    const logger = winston.createLogger({
+      level: "info",
+      transports: [new winston.transports.Console()]
+    });
+    networker = new NetworkerMock();
+    priceFeed = new ETHVIXPriceFeed(logger, false, networker, getTime, minTimeBetweenUpdates);
+    inversePriceFeed = new ETHVIXPriceFeed(logger, true, networker, getTime, minTimeBetweenUpdates);
+  });
+
+  describe("After an initial update has been performed", () => {
+    beforeEach(async () => {
+      networker.getJsonReturns = new Array(3).fill(historicalResponse);
+    });
+
+    describe("Price discovery", () => {
+      it("can return the latest ethVIX price", async () => {
+        await priceFeed.update();
+        assert.equal(priceFeed.getCurrentPrice().toString(), "70200000");
+      });
+
+      it("can return the latest iethVIX price", async () => {
+        await inversePriceFeed.update();
+        assert.equal(inversePriceFeed.getCurrentPrice().toString(), "142440000");
+      });
+    });
+
+    describe("Historical price discovery", () => {
+      it("can return the historical ethVIX price at a specific time", async () => {
+        const { timestamp, vix } = historicalResponse[1];
+        await priceFeed.update();
+        const historicalPrice = await priceFeed.getHistoricalPrice(moment.utc(timestamp).valueOf());
+        assert.equal(historicalPrice.toString(), parseFixed(vix, 6).toString());
```

Comment:
all these cases where you are asserting values on the price feed would need to be changed to `1e18` scalled numbers. (i.e `toWei` from web3)